### PR TITLE
feat(layout): add popover for new journal entry

### PR DESCRIPTION
**Pull request by byhow**
I was working on the `Button` stories prior, so I happen to refactor that page as well. Others are in the `Tabs` story.

re: #4486

### CODE REVIEW

File: src/stories/components/Tabs/JournalTabsGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames'
+import React, { useState } from 'react'
+import { useIntl } from 'react-intl'
+
+import styles from '../../../views/ArticleDetail/AuthorSidebar/Tabs/styles.module.css'
```
Comment (robertu7) [inline src/stories/components/Tabs/JournalTabsGroup.tsx:5]: move `~/views/ArticleDetail/AuthorSidebar/Tabs` to `~/components` and make it reusable?

**Review by robertu7 — APPROVED** [review]

File: src/stories/components/Tabs/JournalTabsGroup.tsx (hunk 1 above)
Comment (byhow) [reply src/stories/components/Tabs/JournalTabsGroup.tsx:5]: The logic in `~/views/ArticleDetail/AuthorSidebar/Tabs` is pretty coupled actually, see https://github.com/thematters/matters-web/blob/25c68f503654a379071ea09619096d69bdc41985/src/views/ArticleDetail/AuthorSidebar/Tabs/index.tsx#L46

Maybe in the future when we figured those conditions can be refactored out of the component we can make the move. I can add a TODO for now

Comment (robertu7) [reply src/stories/components/Tabs/JournalTabsGroup.tsx:5]: should abstract those UI elements and make it reusable without specific business logic.

Comment (robertu7) [reply src/stories/components/Tabs/JournalTabsGroup.tsx:5]: > I can add a TODO for now
looks good for now. we can do it when integrating it into following page.

**Pull request merged**